tutorials/simulators/OpenFOAM.md

```diff
@@ -360,25 +262,109 @@ much more.
 
 ### Step 4: Enhancing Performance with MPI Cluster
 
-As you have experienced, simulations can take a long, long time. To further
-reduce runtime we can change our machine configuration to a MPI cluster
-with two machines:
+Since we now want to scale our simulation into multiple machines we can no longer
+let Openfoam take care of our MPI configuration. We need to do it manually to
+let our API know how to handle MPI.
 
-```python
-mpi_cluster = inductiva.resources.MPICluster(
-                  machine_type="c3d-highcpu-360",
-                  data_disk_gb=300,
-                  num_machines=2)
-mpi_cluster.start()
+The first thing we need to do is move from running `Allrun` into setting the
```

Comment:
The first thing we need to do is replace running `Allrun` with manually setting the commands with the correct MPI configuration.